package-overrides/github/tinymce/tinymce-dist@4.0.0.json

```diff
@@ -0,0 +1,258 @@
+{
+  "format": "global",
+  "main": "tinymce",
+  "shim": {
+    "tinymce.min": {
+      "exports": "tinymce"
+    },
+    "tinymce": {
+      "exports": "tinymce"
+    },
+    "plugins/advlist/plugin": {
+      "imports": "tinymce",
```

Comment:
`imports` is not a valid shim property. I think we might have used `imports` once though in the past and it may still be supported for backwards compat, but best to just use `deps`.